proposals/threads/Overview.md

```diff
@@ -590,6 +598,8 @@ instr ::= ...
         | 0xFE 0x01 m:memarg32  =>  i32.atomic.wait m
         | 0xFE 0x02 m:memarg64  =>  i64.atomic.wait m
 
+        | 0xFE 0x0F 0x00        =>  atomic.fence
```

Comment:
There's really no reason why this should be 0x0F and not 0x03.  It's already part of the 0xFE 0x0_ group, and we might as well pack each group densely.

Reply:
I was thinking that if we added something like a fork instruction, it would be nice to still have fence be closest to the load/store definitions. I agree it's not very strong reason though.

I've changed it to 0x03 now.

Reply:
Reasonable people can disagree about this, but personally I think it doesn't gain us anything to try to group the encodings of operations that have related functionality.  All of us are going to implement dispatch by lookup tables anyway, and over time there will be enough encoding exceptions that any system will come to seem non-coherent.